src/lib/Interface.js

```diff
@@ -191,6 +191,26 @@ export default class Interface {
     );
   }
 
+  buildInitialModal() {
+    return el('dialog#cconsent-modal',
+      el('div.ccm__content',
+        el('div.ccm__content__heading',
+          el('h2#ccm__content__title', Language.getTranslation(window.CookieConsent.config, window.CookieConsent.config.language.current, 'modalMainTitle')),
+          el('p',
+            Language.getTranslation(window.CookieConsent.config, window.CookieConsent.config.language.current, 'modalMainText'),
+            (window.CookieConsent.config.modalMainTextMoreLink) ? el('a', { href: window.CookieConsent.config.modalMainTextMoreLink, target: '_blank', rel: 'noopener noreferrer' }, Language.getTranslation(window.CookieConsent.config, window.CookieConsent.config.language.current, 'learnMore')) : null
+          ),
+        ),
+        el('div.ccm__footer',
+          el('button.consent-decline', Language.getTranslation(window.CookieConsent.config, window.CookieConsent.config.language.current, 'initModalBtnDisagree')),
+
+          el('button.ccm__edit', Language.getTranslation(window.CookieConsent.config, window.CookieConsent.config.language.current, 'initModalBtnOptions')),
+          el('button.consent-give', Language.getTranslation(window.CookieConsent.config, window.CookieConsent.config.language.current, 'initModalBtnAgree')),
+        )
+      ), { 'aria-labelledby': 'ccm__content__title', 'aria-hidden': 'true' }
+    );
+  }
+
```

Comment:
This creates a base dark overlay, and when opening the settings another overlay is added making it darker.
Let's make sure there is only one overlay.

Reply:
Both modals are shown/hidden as needed, so the two overlays are not shown together causing the darkening effect